src/main/resources/public/js/src/projectSettings/tabViews/autoAnalysis/autoAnalysisTabView.js

```diff
@@ -69,11 +70,31 @@ define(function (require) {
                     }
                     return Localization.project.generateIndex;
                 }
+            },
+            isIndexBtnDisabled: {
+                deps: ['indexing_running'],
+                get: function (indexing_running) {
+                    if (!this.registryModel.get('services').ANALYZER) {
+                        return true;
+                    } else if (indexing_running) {
+                        return true;
+                    }
+                    return false;
+                }
+            },
+            indexBtnTitle: {
+                get: function () {
+                    if (!this.registryModel.get('services').ANALYZER) {
+                        return Localization.project.noAutoAnalysisService;
+                    }
+                    return '';
```

Comment:
return !this.registryModel.get('services').ANALYZER ? Localization.project.noAutoAnalysisService : '';